Sources/XMLCoder/Decoder/DecodingErrorExtension.swift

```diff
@@ -23,7 +23,7 @@ internal extension DecodingError {
         let description = "Expected to decode \(expectation) but found \(_typeDescription(of: reality)) instead."
         return .typeMismatch(expectation, Context(codingPath: path, debugDescription: description))
     }
-
+    
```

Comment:
This looks like trailing whitespaces on a blank line, was this committed by mistake or added by SwiftFormat? In either case it would be great to avoid these